### DISCUSSION TITLE
Local Script Subscriptions?

### DISCUSSION THREAD

**Discussion by catalinauser** [ViennaRSS/vienna-rss#1738] [Q&A]
Can Vienna be made to "subscribe" to a local shell script, for the purpose of modifying incoming feeds to my taste (or in the case that links in the feed are broken and the site operator is unresponsive to requests to fix)?

**Comment by matthewruzzi** [comment]
Similar to https://rssguard.readthedocs.io/en/stable/features/scraping.html
